src/equiv.js

```diff
@@ -103,13 +101,12 @@ const callbacks = {
   },
 
   array (a, b) {
-    const len = a.length;
```

Comment:
I believe this was meant to account for cases where length is a (custom) getter rather than a plain property or (native) array accessor. Having said that, for custom objects we shouldn't be using this function so probably fine? Just leaving here as FYI, this is not a problem.

Reply:
Yes I think this shouldnt be a problem due to a typeOf here: https://github.com/qunitjs/qunit/blob/main/src/equiv.js#L282